backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3Service.java

```diff
@@ -0,0 +1,8 @@
+package de.pretrendr.businesslogic;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface S3Service {
+	public Map<String, Integer> getWorkdCountMapFromBucketName(String bucket_name) throws IOException;
```

Comment:
this should probably be getWorkCount... right?

Reply:
Typo fixed.